docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  btfs-build-binaries:
+    build:
+      context: .
+      dockerfile: Dockerfile.unit_testing
+    volumes:
+      - ../btfs-binary-releases:/btfs-binary-releases
```

Comment:
Can you help to fill me some background info here -- why copy previous container's volume to the new?

Reply:
when running the docker container to build the binaries, we need to share a directory on the jenkins slave so the binaries built by the container will persist and can be uploaded to the s3 bucket as the last step of the jenkins job.
when docker-compose runs .`/package.sh` in the go-btfs dir, this script places the built binaries in the btfs-binary-releases directory that is at the same level as the go-btfs dir.
we dont need to necessarily copy the files from ../btfs-binary-releases into the container but it needs to be accessable by the container as a place to save the newly generated binaries.

Reply:
I see, thanks for the info.